account_avatax_website_sale/README.rst

```diff
@@ -0,0 +1,3 @@
+=========================================
+Avalara Avatax Connector for eCommerce
+=========================================
```

Comment:
Can you please add a README subdirectory?
Have a peek at the other modules to see how.